.github/workflows/enforce-pr-labels.yml

```diff
@@ -0,0 +1,18 @@
+name: Enforce labels on Pull Request
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: mheap/github-action-required-labels@v5
+        with:
+          mode: exactly
+          count: 1
+          labels: "[Type] Accessibility, [Type] Automated Testing, [Type] Bug, [Type] Build Tooling, [Type] Code Quality, [Type] Copy, [Type] Developer Documentation, [Type] Enhancement, [Type] Experimental, [Type] New API, [Type] Other, [Type] Performance, [Type] Project Management, [Type] WP Core Ticket"
+          add_comment: true
+          message: "## ⚠️ Type of PR label error\n To merge this PR, it requires {{ errorString }} {{ count }} label indicating the type of PR. Other labels are optional and not being checked here. \n- **Type-related labels to choose from**: {{ provided }}.\n- **Labels found**: {{ applied }}."
```

Comment:
Feedback on the copy is welcome!

Reply:
I'd say "To merge this PR, please add at least one label indicating the type of PR".

Reply:
> at least

One of the goals here is to avoid having more than one label indicating the type of PR, as we should avoid conflicting types. If we need more than one category, it means we don't have the right categories 😅

Reply:
I get the three types Bug, Enhancement, Documentation. 
What is the rational behind making "good first issue" a suggestion for a PR?

Reply:
@bph The screenshot is from a test repository containing GitHub's default labels and demonstrates how it would work and look, but the label settings are different. I did not make a 1:1 mapping with Gutenberg's labels in that test repo; the labels that would be enforced are the ones listed in this PR description.